app.json

```diff
@@ -48,6 +50,11 @@
         "resizeMode": "contain",
         "backgroundColor": "#191015"
       }
+    },
+    "extra": {
+      "eas": {
+        "projectId": "95f68b2a-7930-42f8-bd69-10519da86afa"
```

Comment:
this is currently under my expo account, we should update this to an IR specific value in the future

Reply:
Yeah we'll have to set up the `owner` property to target the IR Expo Org (where we would then add our individual accounts). This will allow the team to run builds and not the public against our account.

Reply:
I just accepted the invite and `yarn build:ios` worked for me, so all good here. Resolving